src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -192,17 +195,84 @@ export abstract class InstructorSessionBasePageComponent {
   }
 
   /**
-   * Copies the feedback session.
+   * Creates list of copy session requests from params
+   * @param model the source session model
+   * @param result the result of the copy session modal
+   * @returns the list of copy session requests
    */
-  copySession(model: SessionsTableRowModel, result: CopySessionResult): void {
-    this.copyFeedbackSession(model.feedbackSession, result.newFeedbackSessionName, result.copyToCourseId)
-        .subscribe((createdSession: FeedbackSession) => {
-          this.navigationService.navigateWithSuccessMessage(
-              this.router,
-              '/web/instructor/sessions/edit',
-              'The feedback session has been copied. Please modify settings/questions as necessary.',
-              { courseid: createdSession.courseId, fsname: createdSession.feedbackSessionName });
-        }, (resp: ErrorMessageOutput) => { this.statusMessageService.showErrorToast(resp.error.message); });
+  copySessionFromRowModel(model: SessionsTableRowModel, result: CopySessionResult): Observable<FeedbackSession>[] {
+    const copySessionRequests: Observable<FeedbackSession>[] = [];
+    this.failedToCopySessions = {};
```

Comment:
This method (building the requests) should not be concerned with resetting this field. The field needs to be reset when there is a fresh request to copy sessions, not when there is a need to build the HTTP requests.